editor/Editor.tsx

```diff
@@ -1191,14 +1194,23 @@ function Editor(props: RouteComponentProps) {
                                     <div
                                         style={{
                                             width: isResponsive && screenSize?.width ? screenSize.width : '100%',
+                                            position: 'relative',
                                             height:
                                                 isResponsive && screenSize?.height
                                                     ? screenSize.height
                                                     : 'calc(100% - 50px)',
                                             background: isResponsive ? 'white' : 'none'
                                         }}
                                     >
-                                        <GoslingComponent ref={gosRef} spec={goslingSpec} theme={'light'} />
+                                        <GoslingComponent
+                                            ref={gosRef}
+                                            spec={goslingSpec}
+                                            theme={'light'}
+                                            visualized={() => {
```

Comment:
We need to review if this callback function makes sense. Gosling used to have "compiled()" callback function that did the similar thing.

Reply:
To track this: #1171